packages/sozluk-content/terms/dba.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: DBA - Database Administrator
+tags:
```

Comment:
I think we should use kebab-case for the tags

eg: database-administration, database-management

Reply:
Oh, I totally forgot about that. Will send a commit to fix this, thanks!